classes/external/load_packages.php

```diff
@@ -0,0 +1,91 @@
+<?php
+// This file is part of the QuestionPy Moodle plugin - https://questionpy.org
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace qtype_questionpy\external;
+
+use external_api;
+use external_function_parameters;
+use external_single_structure;
+use external_value;
+use moodle_exception;
+use qtype_questionpy\api\api;
+use qtype_questionpy\package\package;
+
+/**
+ * This service loads QuestionPy packages from the application server into the database.
+ * Before doing so, it removes previously loaded packages from the database - packages that were uploaded by a trainer
+ * will not be removed.
+ *
```

Comment:
Moodlecheck will, dass die erste Zeile alleine steht
```suggestion
 * This service loads QuestionPy packages from the application server into the database.
 * 
 * Before doing so, it removes previously loaded packages from the database - packages that were uploaded by a trainer
 * will not be removed.
 *
```